### DISCUSSION TITLE
Do we have a roadmap somewhere?

### DISCUSSION THREAD

**Discussion by HarryYC** [ZuodaoTech/everyone-can-use-english#522] [Ideas]
Having a roadmap would be super useful. It lays out what's coming up for the app, like new features or bugs. 
It's like having a map for where the app is going, so I can pick the best path to contribute when I have time.

It seems like Github already has a good feature for this [link](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/changing-the-layout-of-a-view#changing-the-project-layout). Maybe we can leverage that.

[real life example](https://github.com/orgs/dotnet/projects/64/views/12)

**Comment by an-lee** [comment]
Great point. I think that would be helpful.

However, I'm not the owner of this repository, so I can't create a project here.

ping @xiaolai
